common/logging/core/src/main/java/io/apiman/common/logging/impl/DoubleLogger.java

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright 2021 Scheer PAS Schweiz AG
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.apiman.common.logging.impl;
+
+import io.apiman.common.logging.IApimanLogger;
+
+/**
+ * Combine two loggers together!
+ *
+ * Can be useful for testing and if logging output should go to somewhere in addition to the logging
+ * subsystem (e.g. response body).
+ *
+ * @author Marc Savy {@literal <marc@blackparrotlabs.io>}
+ */
+public class DoubleLogger implements IApimanLogger {
+
+    private final IApimanLogger loggerA;
```

Comment:
Are these two loggers for logging to stdout (console) and one to log into a file?

Reply:
Logging out to both console and to the response body of the request (see: https://github.com/msavy/apiman/blob/add-jboss-logging-to-wf-eap-distros/manager/api/rest-impl/src/main/java/io/apiman/manager/api/rest/impl/SystemResourceImpl.java#L177-L244)

This provides a backwards compatible mechanism of achieving this (so we get both to the logging framework and to the alternative source).